lang/rust/rust-values.mk

```diff
@@ -52,8 +52,12 @@ endif
 
 # ARM Logic
 ifeq ($(ARCH),arm)
+  ifneq ($(CONFIG_arm_v6),y)
+    RUSTC_TARGET_ARCH:=$(subst arm,armv5te,$(RUSTC_TARGET_ARCH))
+  endif
+
   ifeq ($(CONFIG_arm_v7),y)
-    RUSTC_TARGET_ARCH:=$(subst arm,armv7,$(RUSTC_TARGET_ARCH))
+    RUSTC_TARGET_ARCH:=$(subst armv5te,armv7,$(RUSTC_TARGET_ARCH))
```

Comment:
```makefile
  ifeq ($(CONFIG_arm_v6)$(CONFIG_arm_v7),)
    RUSTC_TARGET_ARCH:=$(subst arm,armv5te,$(RUSTC_TARGET_ARCH))
  else ifeq ($(CONFIG_arm_v7),y)
    RUSTC_TARGET_ARCH:=$(subst arm,armv7,$(RUSTC_TARGET_ARCH))
  endif
```

this might look bit more clear, what do you think?

Reply:
You're right, updated